staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -104,7 +104,8 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope *RequestSc
 				trace.Step("Decoded delete options")
 
 				ae := request.AuditEventFrom(ctx)
-				audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
+				objGVR := schema.GroupVersionResource{Group: scope.Kind.Group, Version: scope.Kind.Version, Resource: scope.Kind.Kind}
```

Comment:
It seems a good catch, but this modification changes the Resource/subresource original father-son relations, need other's approval   /cc @liggitt @piosz